paymentsheet/src/test/java/com/stripe/android/customersheet/data/CustomerAdapterDataSourceTest.kt

```diff
@@ -0,0 +1,95 @@
+package com.stripe.android.customersheet.data
+
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.customersheet.CustomerAdapter
+import com.stripe.android.customersheet.ExperimentalCustomerSheetApi
+import com.stripe.android.customersheet.FakeCustomerAdapter
+import com.stripe.android.paymentsheet.model.SavedSelection
+import com.stripe.android.testing.PaymentMethodFactory
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+
+@OptIn(ExperimentalCustomerSheetApi::class)
+class CustomerAdapterDataSourceTest {
+    @Test
+    fun `on retrieve payment methods, should completely successfully from adapter`() = runTest {
```

Comment:
should this read "complete" instead of completely?

Reply:
Yup!